10.9-libcxx/stable/main/finkinfo/sci/netcdf-c15.info

```diff
@@ -0,0 +1,136 @@
+Package: netcdf-c15
+Version: 4.6.3
+Revision: 1
+BuildDependsOnly: true
+Maintainer: Remko Scharroo <remkos@users.sourceforge.net>
+
+Depends: %n-shlibs (= %v-%r)
+BuildDepends: doxygen, fink-package-precedence, hdf5.100.v1.10, libcurl4, szip
+Conflicts: <<
+	netcdf-absoft,
+	netcdf, netcdf7,
+	netcdf-c7,
+	netcdf-c11,
+	netcdf-c13,
+	netcdf-c15
```

Comment:
Is this correct?  c15 conflicts with itself?

Reply:
The fink .info processor automatically removes "self" from Conflicts and Replaces lists. That's so you can have a bunch of libversions with identical C/R to reduce differences among the .info and possible typos if you were to have to remember to exclude the correct one from each manually.

Reply:
According to the [reference manual](http://www.finkproject.org/doc/packaging/reference.php?phpLang=en)
<pre>
If a package is listed in its own Conflicts, it will be (silently) removed from that list.
</pre>
It is just practical to do it that way, because then you can copy this to all other (older) versions the same way.